service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang3.ObjectUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.util.CollectionUtils;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import static java.util.stream.Collectors.toList;
+
+public class EmailNotificationHelper {
+
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    private static final String COMMA = ",";
+
+    private EmailNotificationHelper() {
+    }
+
+    public static String buildSubjectLine(final CaseDetails caseDetails) {
+        final List<Respondent> respondents = getRespondents1(caseDetails);
+
+        final String lastName = (StringUtils.isNotBlank(getRespondent1Lastname(respondents))
```

Comment:
I would prefer static method imports, just isNotBlank etc

Reply:
yeah, it could be that. Left this way as it gives us visibility as to where the particular method is coming from. do we have a convention for this?